roles/matrix-conduit/templates/conduit/conduit.toml.j2

```diff
@@ -0,0 +1,52 @@
+# =============================================================================
+#  This is the official example config for Conduit.
+#  If you use it for your server, you will need to adjust it to your own needs.
+#  At the very least, change the server_name field!
+# =============================================================================
+
+
+[global]
+# The server_name is the pretty name of this server. It is used as a suffix for user
+# and room ids. Examples: matrix.org, conduit.rs
+
+# The Conduit server needs all /_matrix/ requests to be reachable at
+# https://your.server.name/ on port 443 (client-server) and 8448 (federation).
+
+# If that's not possible for you, you can create /.well-known files to redirect
+# requests. See
+# https://matrix.org/docs/spec/client_server/latest#get-well-known-matrix-client
+# and
+# https://matrix.org/docs/spec/server_server/r0.1.4#get-well-known-matrix-server
+# for more information
+
+server_name = "{{ matrix_domain }}"
+
+# This is the only directory where Conduit will save its data
+database_path = "/var/lib/matrix-conduit/"
+database_backend = "rocksdb"
+
+# The port Conduit will be running on. You need to set up a reverse proxy in
+# your web server (e.g. apache or nginx), so all requests to /_matrix on port
+# 443 and 8448 will be forwarded to the Conduit instance running on this port
+# Docker users: Don't change this, you'll need to map an external port to this.
+port = {{ matrix_conduit_port_number }}
+
+# Max size for uploads
+max_request_size = {{ matrix_conduit_max_request_size }}
+
+# Enables registration. If set to false, no users can register on this server.
+allow_registration = {{ matrix_conduit_allow_registration | to_json | replace("\"", "") }}
+
+allow_federation = {{ matrix_conduit_allow_federation | to_json | replace("\"", "") }}
+
+# Enable the display name lightning bolt on registration.
+enable_lightning_bolt = {{ matrix_conduit_enable_lightning_bolt | to_json | replace("\"", "") }}
+
+trusted_servers = {{ matrix_conduit_trusted_servers }}
```

Comment:
```suggestion
trusted_servers = {{ matrix_conduit_trusted_servers | to_json }}
```

This is a list in `defaults/main.yml` and in [conduit-example.toml](https://gitlab.com/famedly/conduit/-/blob/0b926c2a31deff57a3526dd75d8c08775b02241a/conduit-example.toml#L46), so I suppose we can just use ` | to_json` here and yield a proper list.

Maybe it happen implicitly too, but it's better to be explicit.

Reply:
Heh, as you can probably see, I know next to nothing about Jinja templating, and I have only a basic working knowledge of Ansible.  Thanks for the suggestions!

I implemented everything above.  Now running through a full end-to-end test on my system before I commit.